torch_glow/src/FusingOptimizer.cpp

```diff
@@ -35,6 +35,12 @@ graph(%input, %weight, %bias, %4):
   %output = aten::matmul(%input, %weight_t)
   %res = aten::add_(%output, %bias, %4)
   return (%res))IR";
+  std::string mm_add_pattern = R"IR(
+graph(%input, %weight, %bias, %4):
```

Comment:
weird indent?

Reply:
Actually not, it is hardcoded IR pattern, and can only use this indent.